cpp/src/Ice/EndpointFactoryManager.cpp

```diff
@@ -138,7 +138,7 @@ IceInternal::EndpointFactoryManager::create(const string& str, bool oaEndpoint)
             // and ask the factory to read the endpoint data from that stream to create
             // the actual endpoint.
             //
-            OutputStream bs(_instance.get(), Ice::currentProtocolEncoding);
+            OutputStream bs;
```

Comment:
I think the previous version was bogus. Doesn't really matter since the 1.0 and 1.1 encoding is identical for all endpoints as far as I can tell.